INN/umbrella-caribbean
# Make sure Largo Recent Posts looks ok on article bottom

## Changes

This pull request makes the following changes:

- Styles the Largo Recent Posts widget as a 3 column grid in the Article Bottom widget area

Desktop:
![Screen Shot 2019-12-02 at 5 07 55 PM](https://user-images.githubusercontent.com/18353636/69999432-a5a1b400-1526-11ea-9140-e9ca6745e86f.png)

Mobile:
![screencapture-caribbean-test-2019-12-02-17_08_00](https://user-images.githubusercontent.com/18353636/69999431-a5a1b400-1526-11ea-815f-394666344bd6.png)

## Why



For #25

## Testing/Questions

Features that this PR affects:

- Largo Recent Posts on article bottom


Questions that need to be answered before merging:


Steps to test this PR:

1. Add Largo Recent Posts widget to Article Bottom widget area
2. Make sure it looks ok on desktop and mobile
